Configuration/DataProcessing/python/Merge.py

```diff
@@ -52,6 +55,7 @@ def mergeProcess(*inputFiles, **options):
         process.add_(Service("DQMStore"))
     else:
         process.source = Source("PoolSource")
+        process.source.bypassVersionCheck = CfgTypes.untracked.bool(bypassVersionCheck)
```

Comment:
@srimanob this is not the final version we have in master and 10_6_X, although it is consistent with 9_4_X and 8_0_X. As far as I can see they are anyway equivalent as final functionality

Reply:
@fabiocos Thanks for checking. This is following the discussion we had before that we turn on the bypassversioncheck by default for the release that will be used for UL-HLT. 


If we follow the master, we need to handle at the cmsDriver level.

Reply:
@srimanob yes, just let's keep in mind that while new 8_0_X and 9_4_X will now be used basically only for UL, 10_2_X is still being used for other purposes, and this change will affect not only UL but whatever other production updated to this version, there is no special UL branch

Reply:
@fabiocos 
This option can't config from McM side as McM don't touch the Merge process. We also don't have the switch in wmcore too. What we agreed on 
https://github.com/dmwm/WMCore/issues/9054
is to set the default value to true by default. The original PR was done in such a way that we can make it configurable in future.

Reply:
@srimanob right, thank you for the reminder, the important things is that we are all aware of the implication